src/Toolkit/Html.php

```diff
@@ -133,9 +133,11 @@ public static function a(string $href, $text = null, array $attr = []): string
      *                           Key-value array: A list of attributes will be generated. Don't pass a second argument in that case.
      * @param mixed $value If used with a `$name` string, pass the value of the attribute here.
      *                     If used with a `$name` array, this can be set to `false` to disable attribute sorting.
+     *                     * @param string|null $before An optional string that will be prepended if the result is not empty
```

Comment:
```suggestion
     * @param string|null $before An optional string that will be prepended if the result is not empty
```

With these `$before` and `$after` arguments I'm not sure if we should really support `null`. I know we already do for the `attr()` helper, but maybe that should instead use `$before ?? ''` when calling this method. Otherwise we would cement those sloppy `null` arguments even more.

If we decide we do want to keep `null` support, that `?? ''` needs to happen inside this method to avoid PHP warnings in the future.

Reply:
I don't really have an opinion here to be honest.

Reply:
@bastianallgeier do you have an opinion? Guess this is the last outstanding change (plus your review then)

Reply:
Thinking about it again, I'm fine with either solution. Only supporting `string` would be cleaner, but inconsistent. So maybe we should leave it basically like it is. With one small change: [This line here](https://github.com/getkirby/kirby/pull/4018/files#diff-9541b56c0625a5edd10a99c7249273140bb08e4b53d3cf4ce1f696034c997c78R161) should use `?? ''` to ensure that always a string is concatenated.

Reply:
Let's leave it then. I would also leave the mentioned line as is to make it easier to read. `null` will always converted to an empty string in PHP. https://www.php.net/manual/en/language.types.string.php#language.types.string.casting